### PR TITLE
got basic deep linking functionality working

### PR DESCRIPTION
**Pull request by aryan-bhardwaj**
Able to open specific pages in app with params using link in browser

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+          <Navigation></Navigation>
+        </NavigationContainer> 
+      </SafeAreaProvider>
+    </GluestackUIProvider>
+  );
+}
+
+{/* <Box width="100%" height="100%" justifyContent="center" alignItems="center">
+		<Text>{data ? JSON.stringify(data) : "App not opened from link"}</Text>
+		</Box> */}
```
Comment (AydanPirani) [inline App.tsx:60]:
```suggestion
```

File: App.tsx [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
+	}
+  }
+
+  const handleDeepLink = (event: LinkingEvent) => {
+    let parsedData = Linking.parse(event.url);
+    setData(parsedData);
+  };
+
+  useEffect(() => {
+
```
Comment (AydanPirani) [inline App.tsx:34]:
```suggestion

```

**Review by AydanPirani — APPROVED** [review]
lgtm! just 1-2 more changes, then we can merge :)

**Review by divyack2 — APPROVED** [review]
lgtm

**Pull request merged**